src/matlab2tikz.m

```diff
@@ -1519,33 +1539,29 @@ case guitypes()
     % a reasonable default.
     matlabDefaultNumMinorTicks = 3;
     if ~isempty(ticks)
-        options = opts_add(options, ...
-            [axis,'tick'], sprintf('{%s}', ticks));
+        options = opts_add(options, [axis,'tick'], sprintf('{%s}', ticks));
```

Comment:
True, but in nearly all the rest of the code there is. So you generally have the pattern
```
         options = opts_add(options, ...
             [axis,'ticklabels'], sprintf('{%s}', tickLabels));
```

Therefore, in total it is more consistent to write it that way, so that  the actual command is on a single line. But this is advanced nitpicking :smiling_imp: